src/backend/aspen/workflows/nextstrain_run/create_phyloruns.py

```diff
@@ -245,39 +241,72 @@ def get_template_args_for_focal_group(
     return None
 
 
+def get_pathogen_db_objects(db: Session, split_client: SplitClient, pathogen: str):
+    pathogen_obj = (
+        db.execute(sa.select(Pathogen).filter(Pathogen.slug == pathogen))
+        .scalars()
+        .one()
+    )
+    default_repository = split_client.get_pathogen_treatment(
+        "PATHOGEN_public_repository", pathogen_obj
+    )
+    import logging
+
+    logger = logging.getLogger(__name__)
+    logger.setLevel(logging.DEBUG)
+    logger.debug(f"pathogen: {pathogen_obj.slug}")
+    logger.debug(f"split_repo_value for pathogen: {default_repository}")
```

Comment:
Did you mean to leave this debug code in here? We can at least move the import to the top.

Reply:
no i did not haha. long day, i'll get rid of it